c2corg_api/models/userdata.py

```diff
@@ -0,0 +1,58 @@
+import bcrypt
+
+from sqlalchemy import (
+    Column,
+    Integer,
+    String
+    )
+
+from c2corg_api.models import Base, users_schema
+
+
+class UserData(Base):
+    """
+    Class containing the users' private and authentication data.
+    """
+    __tablename__ = 'userdata'
+    __table_args__ = {"schema": users_schema}
+    __mapper_args__ = {
+        'polymorphic_identity': 'ud'
+    }
+
+    document_id = Column(Integer, primary_key=True)
```

Comment:
No. It's different from the User documents.
But the user id should be related to the User document id.

By the way I'm a bit annoyed: that would make sense to rename this model to User but it would conflict with the future User document model. Or maybe we could name the user document as Profile for instance?